docs/reference/query-languages/esql/esql-advanced-commands.md

```diff
@@ -0,0 +1,7 @@
+# Advanced commands
+
+You can perform complex data transformations and enrichment with {{esql}} with the following commands:
+
+* [Extract data with `DISSECT` and `GROK`](/reference/query-languages/esql/esql-process-data-with-dissect-grok.md): Learn how to extract and transform structured data from unstructured text using the `DISSECT` and `GROK` commands.
+* [Augment data with `ENRICH`](/reference/query-languages/esql/esql-enrich-data.md): Learn how to combine data from different indices using the `ENRICH` command.
+* [Join data with `LOOKUP JOIN`](/reference/query-languages/esql/esql-lookup-join.md): Learn how to join data from different indices using the `LOOKUP JOIN` command.
```

Comment:
nit: the links already say "with `DISSECT` and `GROK`", which is later repeated in the description "... using the `DISSECT` and `GROK` commands".

Maybe it's nicer like this?
```suggestion
* [Extract data with `DISSECT` and `GROK`](/reference/query-languages/esql/esql-process-data-with-dissect-grok.md): Learn how to extract and transform structured data from unstructured text.
* [Augment data with `ENRICH`](/reference/query-languages/esql/esql-enrich-data.md): Learn how to combine data from different indices.
* [Join data with `LOOKUP JOIN`](/reference/query-languages/esql/esql-lookup-join.md): Learn how to join data from different indices.
```